util/util_test.go

```diff
@@ -181,3 +181,42 @@ func TestReplaceEvalWithMap(t *testing.T) {
 	testReplaceEvalWithMap(t, "eval(rule1) || eval(rule2)", nil, "eval(rule1) || eval(rule2)")
 	testReplaceEvalWithMap(t, "eval(rule1) || eval(rule2) && c && d", nil, "eval(rule1) || eval(rule2) && c && d")
 }
+
+func testCacheGet(t *testing.T, c *LRUCache, key string, value interface{}, ok bool) {
+	v, o := c.Get(key)
+	if v != value || o != ok {
+		t.Errorf("Get(%s): (%s, %t) supposed to be (%s, %t)", key, v, o, value, ok)
+	}
+}
+
+func testCachePut(t *testing.T, c *LRUCache, key string, value interface{}) {
+	c.Put(key, value)
+	v, o := c.Get(key)
+	if v != value || o != true {
+		t.Errorf("Put(%s, %s): didn't add value", key, value)
+	}
+}
+
+func testCacheEqual(t *testing.T, c *LRUCache, values []int) {
+	cacheValues := []int{}
+	for _, v := range c.m {
+		cacheValues = append(cacheValues, v.value.(int))
+	}
+
+	if SetEqualsInt(values, cacheValues) == false {
+		t.Errorf("cache values: %d supposed to be %d", cacheValues, values)
+	}
+}
+
+func TestLRUCache(t *testing.T) {
+
```

Comment:
go fmt all the code

Reply:
Running `gofmt -s -w -l .` does not change anything for me.

Reply:
remove empty lines like 212